src/line.rs

```diff
@@ -12,6 +12,16 @@ impl<'src> Line<'src> {
     self.fragments.is_empty()
   }
 
+  pub(crate) fn is_comment(&self) -> bool {
+      match self.fragments.first() {
+          Some(Fragment::Text { token }) => {
+              let lexeme = token.lexeme();
+              lexeme.starts_with("#") && !lexeme.starts_with("#!")
```

Comment:
If this is only called for linewise recipes, then does it need to check for `#!`?

Reply:
It doesn't, but the code can't guarantee that no one will ever try to call `is_comment` in a shebang recipe in the future.

On the other hand, maybe it makes sense to say that, once you're in the body of a linewise recipe, `#!` should be interpreted as a comment that happens to start with a `!` rather than a shebang. Actually I think this is the correct thing to do, because right now just only checks to see if the first line in a recipe is a `#!`, not any other ones. So if the code encounters a `#!` after the first line, it should treat it as a comment that happens to begin with `!`, not as a shebang.